### ISSUE TITLE
Val loss is nan when training the model

### ISSUE THREAD

**Issue by yhong4** [RyanWangZf/transtab#1]
Hello, really appreciate the share of your work.

I got an error when I run the fast_train.ipynb in the examples folder. The val_loss always displays "nan". Is there anything I miss? I don't change any code.
```
epoch: 0, test val_loss: nan
epoch: 0, train loss: 7.0228, lr: 0.000100, spent: 0.7 secs
epoch: 1, test val_loss: nan
epoch: 1, train loss: 6.7598, lr: 0.000100, spent: 1.4 secs
```

**Comment by RyanWangZf** [comment]
Hi, I fix this issue w/ the new commit. Please try to install this package by
``
pip install git+https://github.com/RyanWangZf/transtab.git
``

and run this example again.
